README.md

```diff
@@ -1,18 +1,15 @@
-## ⚠️ Maintenance & support
+## ⚠️ Deprecated
 
-This library continues to be relied upon heavily by Atlassian products, but we are focused on other priorities right now and have no current plans for further feature development or improvements.
+Hey all,
 
-It will continue to be here on GitHub and we will still make critical updates (e.g. security fixes, if any) as required, but will not be actively monitoring or replying to issues and pull requests.
+We are taking the next step in saying thank you and goodbye to our friend `react-beautiful-dnd`.
 
-We recommend that you don’t raise issues or pull requests, as they will not be reviewed or actioned until further notice.
+- ⚠️ We will be soon deprecating `react-beautiful-dnd` on [npm](https://www.npmjs.com/package/react-beautiful-dnd). When we do you will start to get console warnings in your build tools.
```

Comment:
> We will be soon deprecating... on npm

Should we give a specific date for when will be deprecated on NPM triggering `console.warns`. Maybe do it on `On Apr 30, 2025` instead of soon?

Reply:
I am thinking that we deprecate on `npm` soon so that people have an opportunity to migrate before we lock the repo. As of now, the thinking is that this project is "deprecated", but just not archived yet